diff --git a/node_modules/entity-decode/browser.js b/node_modules/entity-decode/browser.js
index cf44a06..24748a0 100644
--- a/node_modules/entity-decode/browser.js
+++ b/node_modules/entity-decode/browser.js
@@ -1,16 +1,12 @@
-/**
- * @see https://github.com/vuejs/vue/commit/a855dd0564a657a73b7249469490d39817f27cf7#diff-c0a2623ea5896a83e3b630f236b47b52
- * @see https://stackoverflow.com/a/13091266/4936667
- */
-
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
 var decoder;
-
-export default function decode(html) {
+function decode(html) {
     decoder = decoder || document.createElement('div');
     // Escape HTML before decoding for HTML Entities
-    html = escape(html).replace(/%26/g,'&').replace(/%23/g,'#').replace(/%3B/g,';');
+    html = escape(html).replace(/%26/g, '&').replace(/%23/g, '#').replace(/%3B/g, ';');
     // decoding
     decoder.innerHTML = html;
-
     return unescape(decoder.textContent);
 }
+exports.default = decode;
